utils/lsof/patches/005-reproducable-build.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/dialects/linux/Makefile b/dialects/linux/Makefile
+index 2bea108..ed8382e 100644
+--- a/dialects/linux/Makefile
++++ b/dialects/linux/Makefile
+@@ -78,8 +78,8 @@ version.h:	FRC
+ 	@echo '#define	LSOF_BLDCMT	"${LSOF_BLDCMT}"' > version.h;
+ 	@echo '#define	LSOF_CC		"${CC}"' >> version.h
+ 	@echo '#define	LSOF_CCV	"${CCV}"' >> version.h
+-	@echo '#define	LSOF_CCDATE	"'`date`'"' >> version.h
+-	@echo '#define	LSOF_CCFLAGS	"'`echo ${CFLAGS} | sed 's/\\\\(/\\(/g' | sed 's/\\\\)/\\)/g' | sed 's/"/\\\\"/g'`'"' >> version.h
++	@echo '#define	LSOF_CCDATE	"'`date -d @${SOURCE_DATE_EPOCH}`'"' >> version.h
```

Comment:
this will fail when building openwrt on macos.

Reply:
@lynxis , would this work better?
https://github.com/mstorchak/packages/commit/ebdb4cc4fa505d6117358775d54e7ffe013b7cf6